requirements.txt

```diff
@@ -20,6 +20,6 @@ django-csp
 django-webpack-loader
 requests_oauthlib
 wand
-beautifulsoup4
+beautifulsoup4<4.6.1
```

Comment:
Could you add a comment noting why this is pinned? It's harder to update dependencies later on if we don't know why we pinned something — IIRC this was a wagtail issue, which might not be obvious.

Reply:
```
ERROR: wagtail 2.8.1 has requirement beautifulsoup4<4.6.1,>=4.5.1, but you'll have beautifulsoup4 4.9.0 which is incompatible.
```

(let me know I should comment this elsewhere)

Reply:
Please add a one-line comment in the requirements file next to the beautiful soup requirement noting that it's pinned because of wagtail 2.8.1

Reply:
Oh that kind of comment, got it

Reply:
Yes, when I reread I saw my request was unclear! You did add a comment like I asked 😁